client/src/components/ChangePasswordModal.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-dialog width="400px">
+    <v-card class="bg-backgroundGrey">
+      <v-img src="/images/Footer-min.jpeg" width="100%" max-height="16" cover></v-img>
```

Comment:
Remove closing tag and close in first tag. Theres a few other places to do this as well